README.md

```diff
@@ -20,18 +22,28 @@ See details at [wiki](https://github.com/miripiruni/CSScomb/wiki/Requirements).
 
 ```
 cd ~/.vim/bundle
-git clone https://github.com/miripiruni/CSScomb-for-Vim.git
+git clone https://github.com/csscomb/vim-csscomb.git
 ```
 
 ### With Vundle
 Add this to .vimrc:
 ```
-Bundle 'git://github.com/miripiruni/CSScomb-for-Vim.git'
+Bundle 'git://github.com/csscomb/vim-csscomb.git'
+```
+
+### With NeoBundle
+Add this to .vimrc:
+```
+NeoBundle 'csscomb/vim-csscomb', {
+        \ 'build': {
+        \     'unix': 'npm install -g csscomb',
+        \     'mac': 'npm install -g csscomb'
+        \ }}
```

Comment:
Personally I have my systems setup to allow sudo-less global npm installs and like my vim plugins to take care of themselves, but I see your point. Removed the build config.